### PR TITLE
fh-cbioportal-stage-1

### PR DESCRIPTION
**Pull request by sitapriyamoorthi**
Adding content to sections. 
All content here is open to review. 
For sections with the tag " Section in progress" note that more content will be upcoming in the next couple of weeks.

### CODE REVIEW

File: _dasldemos/fh-cbio-intro.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+-   what the **Fred Hutch instance of cBioPortal** is and steps to take to upload your data onto the instance.
+-   ways to format your data for upload into cBioPortal.
+
+**What is cBioportal?**
+-----------------------
+**Section in progress**
+-   It is an open-source web-based platform that allows for the visualization, exploration and analysis of cancer genomics data. When you upload data from your study, the portal dynamically creates graphs that allow you to analyze your data based on clinical characteristics.
+-   The public version of this instance can be accessed here: <https://www.cbioportal.org/> and hosts publicly available/published data.
+-   It is interactive and user-friendly
+-   It allows for the easy visualization of both genomic and clinical data
```
Comment (laderast) [inline _dasldemos/fh-cbio-intro.md:22]: I'd be more specific here, such as "cBioPortal enables visualizations that show the interaction between genomic features (such as genes) with clinical covariates. One example is stratifying survival on the presence or absence of particular genomic variants."

File: _dasldemos/fh-cbio-intro.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+-   It is a **local installation** of cBioPortal within the **Fred Hutch infrastructure**.
+-   Looks exactly like the [public instance](https://www.cbioportal.org/) and has all the **same features** as the **public** **instance**.
+-   It is a **secure** **place** to host your data.
+    -   cBioPortal has ben reviewed and approved by the Fred Hutch Information Security team. This permits users to include individually identifying research data such as [Personal Health Information](https://sciwiki.fredhutch.org/datascience/phi/#types-of-data-sets) (PHI) if covered in your studies [IRB](https://sciwiki.fredhutch.org/_datascience/consent_IRB/).
+-   Study **data** is only **visible** to **authorized** personnel approved by the study lead/PI.
+-   Unlike the public instance, you can upload **interim data** onto the instance. This can aid your research by not having you to generate plots and graphs that can often be time consuming.
+
+### How can I access the Fred Hutch instance of cBioportal?
+
+-   Before you are able to access the Fred Hutch instance of cBioPortal you must:
```
Comment (laderast) [inline _dasldemos/fh-cbio-intro.md:42]: Make this into a checklist using `- [ ]` in front of items

File: _dasldemos/fh-cbio-intro.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+### What can I do with cBioportal
+
+How can it be used to facilitate genomic research and visualization.
+
+-   Links : <https://docs.cbioportal.org/user-guide/overview/>
+
+**What is the FH instance?**
+----------------------------
+
+-   It is a **local installation** of cBioPortal within the **Fred Hutch infrastructure**.
```
Comment (laderast) [inline _dasldemos/fh-cbio-intro.md:33]: Think about putting the most important things first:

- The FH cBioPortal is useful for interim visualization of your clinical and genomic data (i.e., before publication)
- Visibility and access to your data is controlled by the PI.
- FH cBioPortal is a secure place to store your data. It has been reviewed and approved by the Fred Hutch Information Security Team

**Review by laderast — CHANGES_REQUESTED** [review]
I added some comments to tighten it up. In general, where you have bullets you need to
prioritize most important information first.

File: _dasldemos/fh-cbio-intro.md (hunk 1 above)
Comment (sitapriyamoorthi) [reply _dasldemos/fh-cbio-intro.md:22]: I have this marked I will update it in a later pull. Thanks

File: _dasldemos/fh-cbio-intro.md (hunk 2 above)
Comment (sitapriyamoorthi) [reply _dasldemos/fh-cbio-intro.md:42]: Done. You will see this in the next draft pull.

File: _dasldemos/fh-cbio-intro.md (hunk 3 above)
Comment (sitapriyamoorthi) [reply _dasldemos/fh-cbio-intro.md:33]: Done

**Comment by sitapriyamoorthi** [comment]
Incorporated the changes suggested by @laderast. Updating and creating a new draft PR.

**Pull request closed without merging**